src/test/cucumber/install_candidate.feature

```diff
@@ -3,62 +3,64 @@ Feature: Install Candidate
   Background:
     Given the internet is reachable
     And an initialised environment
-    And the system is bootstrapped
 
   Scenario: Install a default Candidate
-    Given the candidate "grails" version "2.1.0" is a valid candidate version
+    Given the system is bootstrapped
+    And the candidate "grails" version "2.1.0" is a valid candidate version
     And the default "grails" version is "2.1.0"
-    When I enter "sdk install grails" and answer "Y"
+    When I enter "sdk install grails"
     Then I see "Done installing!"
-    Then the candidate "grails" version "2.1.0" is installed
+    And I do not see "Do you want grails 2.1.0 to be set as default? (Y/n)"
+    And the candidate "grails" version "2.1.0" is installed
 
   Scenario: Install a specific Candidate
+    Given the system is bootstrapped
     And the candidate "grails" version "1.3.9" is available for download
-    When I enter "sdk install grails 1.3.9" and answer "Y"
+    When I enter "sdk install grails 1.3.9"
     Then I see "Done installing!"
-    Then the candidate "grails" version "1.3.9" is installed
+    And I do not see "Do you want grails 2.1.0 to be set as default? (Y/n)"
+    And the candidate "grails" version "1.3.9" is installed
 
   Scenario: Install a Candidate version that does not exist
-    Given the candidate "grails" version "1.4.4" is not available for download
+    Given the system is bootstrapped
+    And the candidate "grails" version "1.4.4" is not available for download
     When I enter "sdk install grails 1.4.4"
     Then I see "Stop! grails 1.4.4 is not available."
 
   Scenario: Install a Candidate version that is already installed
-    Given the candidate "grails" version "1.3.9" is available for download
+    Given the system is bootstrapped
+    And the candidate "grails" version "1.3.9" is available for download
     And the candidate "grails" version "1.3.9" is already installed and default
     When I enter "sdk install grails 1.3.9"
     Then I see "Stop! grails 1.3.9 is already installed."
 
-  Scenario: Install a candidate and select to use it
```

Comment:
It's subtly different because `Install a specific candidate` is the first time a version is installed (so no current default exists). I'm adding the scenario that tests this case too.